### PR TITLE
Poets fixes

### PR DESCRIPTION
**Pull request by dkarnikis**
On script [6_1.sh](https://github.com/binpash/pash/pull/294/files#diff-e9beb2c7b12f38608013f7679b1ea45e7a09f1cbcb671b1a855d28286a7a6170), I don't get the same output as the original UnixforPoets script.

### CODE REVIEW

File: evaluation/benchmarks/poets/6_1.sh [hunk 1]
```diff
@@ -2,46 +2,16 @@
 # tag: trigram_rec
 # set -e
 
-# FIXME: what is this?
 IN=${IN:-$PASH_TOP/evaluation/benchmarks/poets/input/pg/}
 OUT=${OUT:-$PASH_TOP/evaluation/benchmarks/poets/input/output/}
-TER=${TER:-$PASH_TOP/pa.sh}
-
 
 trigrams() {
```
Comment (nvasilakis) [inline evaluation/benchmarks/poets/6_1.sh:8]: One idea would be to completely lay `trigrams` out completely. (cc @angelhof )

**Review by nvasilakis — APPROVED** [review]
Thanks for the cleanup!

**Comment by nvasilakis** [comment]
@dkarnikis Are there any more `poets` scripts that depend on previous/intermediate results and thus are not getting the right input if `IN` points to the Gutenberg files?

**Comment by dkarnikis** [comment]
@nvasilakis I haven't found any other script related to the issue you described

**Review by nvasilakis — APPROVED** [review]
Thank you!

**Pull request merged**